backend/internal/server/apis/schedule/games/manual/games.go

```diff
@@ -0,0 +1,193 @@
+package manual
+
+import (
+	"github.com/gofiber/fiber/v2"
+	"github.com/jak103/powerplay/internal/db"
+	"github.com/jak103/powerplay/internal/models"
+	"github.com/jak103/powerplay/internal/server/apis"
+	"github.com/jak103/powerplay/internal/server/apis/schedule/internal/mapping"
+	"github.com/jak103/powerplay/internal/server/apis/schedule/internal/structures"
+	"github.com/jak103/powerplay/internal/server/services/auth"
+	"github.com/jak103/powerplay/internal/utils/log"
+	"github.com/jak103/powerplay/internal/utils/responder"
+)
+
+func init() {
+	apis.RegisterHandler(fiber.MethodPost, "/schedule/manual/game", auth.Authenticated, handleCreateGame)
+	apis.RegisterHandler(fiber.MethodPost, "/schedule/manual/games", auth.Authenticated, handleCreateGames)
+	apis.RegisterHandler(fiber.MethodGet, "/schedule/manual/games/:id", auth.Authenticated, handleGetGame)
+	apis.RegisterHandler(fiber.MethodGet, "/schedule/manual/games", auth.Authenticated, handleGetGames)
+	apis.RegisterHandler(fiber.MethodPut, "/schedule/manual/games/:id", auth.Authenticated, handleUpdateGame)
+	apis.RegisterHandler(fiber.MethodPut, "/schedule/manual/games", auth.Authenticated, handleUpdateGames)
+	apis.RegisterHandler(fiber.MethodDelete, "/schedule/manual/games/:id", auth.Authenticated, handleDeleteGame)
+	apis.RegisterHandler(fiber.MethodDelete, "/schedule/manual/games", auth.Authenticated, handleDeleteGames)
+}
+
+// basic CRUD operations
+// - the caller is allowed to fine tune the schedule by updating the games
+
+func handleCreateGame(c *fiber.Ctx) error {
+	type Dto struct {
+		Game structures.Game `json:"game"`
```

Comment:
Why not just accept a model.game? thats what the front end will likely have.

Reply:
it would be easiest to talk about this in person.